service/frontend/templates/clusterredirection.tmpl

```diff
@@ -75,27 +79,25 @@ func (handler *clusterRedirectionHandler) {{$method.Declaration}} {
 	}
 	{{- end}}
 
-    {{$policyMethod := "WithDomainNameRedirect"}}
-    {{$domain := printf "%s.GetDomain()" (index $method.Params 1).Name}}
-    {{- if has $method.Name $domainIDAPIs}}
-    token := domainIDGetter(noopdomainIDGetter{})
-    {{- end}}
+	var domainEntry *cache.DomainCacheEntry
```

Comment:
switched to using DomainCacheEntry to avoid a few unnecessary conditional branches based on domain id vs name.